st2common/st2common/models/db/rbac.py

```diff
@@ -76,9 +82,26 @@ class PermissionGrantDB(stormbase.StormFoundationDB):
     permission_types = me.ListField(field=me.StringField())
 
 
+class GroupToRoleMappingDB(stormbase.StormFoundationDB):
```

Comment:
I didn't see a stand alone unit test for this in st2common/unit/test_db_group_role_mapping.py. Did I miss it? I thin k it's important to add one.